Makefile

```diff
@@ -41,9 +41,12 @@ lint:
 	checkconfig -strict -config-path prow/config.gen.yaml -plugin-config prow/plugins.yaml
 	@scripts/check-resource-limits.sh
 
-# this will build the containers and then try to use them to build themselves again, making sure we didn't break docker support
-build-containers: maistra-builder
+# these will build the containers and then try to use them to build themselves again, making sure we didn't break docker support
+build-containers-2.3: maistra-builder_2.3
```

Comment:
let's turn these into dynamic Make targets (like the maistra-builder_XX one above) and then merge this

Reply:
done